pages/visualizations/overview/active_drifting_contributors.py

```diff
@@ -26,7 +26,7 @@
                     disabled=False,
                     n_intervals=1,
                     max_intervals=1,
-                    interval=800,
+                    interval=1500,
```

Comment:
what is this interval change for?

Reply:
(same question for all the following files)

Reply:
so that the front-end pings the web-server less regularly. If there are lots of browsers all pinging the web-server (num_callbacks) times every 800ms that's a lot of traffic. I'm just decreasing the periodicity of that. It doesn't change the user's experience of the speediness of the app from an update perspective.